examples/model/alcap/plot.jl

```diff
@@ -6,5 +6,5 @@
 using GLMakie
 using CarboKitten.Visualization
 
-save("docs/src/_fig/alcaps-alternative.png", summary_plot("data/output/alcap-example.h5"))
```

Comment:
@xyl96 don't change files if you don't understand the consequences. We can't merge with this here.